docs/embedded/getting-started/spembedded-for-vscode.md

```diff
@@ -31,10 +31,17 @@ If you don't already have a tenant, you can get a tenant through the [Microsoft
 ![Install](../images/vsx-images/n2vsx-signin.png)
 
 - Authentication opens a new tab in an external browser to grant permissions
+
+![auth allow extension to open uri](../images/vsx-images/auth-allow-extension-uri.png)
+
 - Select **Accept** on the pop-up window prompting admin consent
```

Comment:
> Review permission requests on the pop-up windows and select **Accept**  to grant admin consent

Let's not tell the user to just accept permissions. Always promote reviewing permissions before accepting anything.